spy_integration_test.go

```diff
@@ -9,24 +9,34 @@ import (
 )
 
 func TestExecuteIntegration(t *testing.T) {
-	var buff bytes.Buffer
-	want := "echo Hello\nHello\n"
-	err := spy.Execute(&buff, "echo Hello")
+	want := struct{ a, b string }{
+		a: "Hello\n",
+		b: "************ START ************\nCOMMAND: echo Hello\n\nOUTPUT:\nHello\n",
+	}
+	var buff, buff2 bytes.Buffer
+	err := spy.Execute(&buff, &buff2, "echo Hello")
 
 	if err != nil {
 		t.Error(err)
 	}
+	got := struct{ a, b string }{
+		a: buff.String(),
+		b: buff2.String(),
+	}
 
-	got := buff.String()
-	if want != got {
-		t.Errorf("Want: %q, got: %q", want, got)
+	if want.a != got.a {
+		t.Errorf("Want: %q, got: %q", want.a, got.a)
+	}
+
+	if want.b != got.b {
+		t.Errorf("Want: %q, got: %q", want.b, got.b)
 	}
 }
 
 func TestExecuteIntegrationInvalid(t *testing.T) {
-	var buff bytes.Buffer
-	want := "echo Hello'\nError parsing command\n"
-	err := spy.Execute(&buff, "echo Hello'")
+	var buff, buff2 bytes.Buffer
```

Comment:
Again perhaps we might name these after what they're pretending to be: `terminal` and `recorder`, maybe.